### DISCUSSION TITLE
How to get rid of `bread` & `package` symbols for javascript projects?

### DISCUSSION THREAD

**Discussion by kinxyo** [starship/starship#5988] [Q&A] [answered]
Whenever I open projects related to Javascript-- React or Nuxt, a `bread` symbol or `bread` & `package` symbols always appear with the prompt.

**React**:
![image](https://github.com/starship/starship/assets/90744941/4a4acbc8-1878-4035-b65f-1c681fe35cab)

**Nuxt** (+ Rust):
![image](https://github.com/starship/starship/assets/90744941/6b1119c3-9a66-45a8-a720-58a9e54a0ba8)

Now, I have my `[package]` `disabled` to `true`, so the package symbol doesn't appear in any other projects (like Go, Rust, Dart, even Nuxt) but only in React. I've went through a lot of config files to change this blasphemous behavior but no luck. It's not even `nodejs` symbol so I don't know what to change here.

Can anybody please help me with this?
[this is my config file](https://github.com/kinxyo/kinxconf/blob/master/Starship/starship.toml)

**Comment by davidkna** [comment, marked as answer]
You can use `starship explain` to show which module a given symbol comes from.
Alongside the `package` module, the 📦-emoji is also your stashed symbol, please try to check if the `git_status` module is responsible for this. As for the bread symbol, please try disabling the `bun` module.
